graphql_test.go

```diff
@@ -3078,6 +3078,14 @@ type helloInput struct {
 	Name string
 }
 
+type helloOutput struct {
+	Name string
+}
+
+func (*helloOutput) Hello() helloOutput {
+	return helloOutput{}
+}
```

Comment:
Why would the `helloOutput` have a `Hello()` method that returns a new instance of `helloOutput`?